docs/TEAM.md

```diff
@@ -0,0 +1,7 @@
+# Team
+
+## Team lead
```

Comment:
Remove either team lead or project manager. It's redundant. 

Also add a spot for the rest of the team members.

Reply:
Team lead and project managers are different roles in companies. The team lead would manage the devs and dev time, the project manager would manage the wider project.

Reply:
So how would this work here? What would the team lead be doing vs the PM?

Reply:
PM's usually are not technical. They get requirements from clients, manage ticket priority, and other stuff like that. In this case, we don't need both. I would go with just a team lead, but they are technically different jobs.

Reply:
Ok Good to know. I'll update stuff elsewhere as well.